.pylintrc

```diff
@@ -38,7 +38,7 @@ load-plugins=linter_plugin
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
-disable=fixme,locally-disabled,abstract-class-not-used
+disable=fixme,locally-disabled,abstract-class-not-used,bad-continuation,too-few-public-methods,no-self-use
```

Comment:
- you don't need `too-few-public-methods` for this PR, please revert
- `no-self-use` is a good indicator that you should use `@staticmethod` or `@classmethod`, please revert or disable locally if you must
- please fix you editor instead of changing `bad-indentation`